_observing-your-data/alerting/monitors.md

```diff
@@ -230,7 +224,7 @@ The maximum number of monitors you can create is 1,000. You can change the defau
     **Note**: Anomaly detection is available only if you are defining a per query monitor.
```

Comment:
@AWSHurneyt Is this still an accurate statement?

Reply:
@vagimeli Yes, this is still accurate.